client/stylesheets/_responsive.scss

```diff
@@ -205,6 +290,27 @@
     // .profileCardLi {
     // 	width: 43% !important;
     // }
+    .searchResultsCardPrice {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `searchResultsCardPrice` should be written in lowercase with hyphens